choreography/src/choreography/Frabrikam.Choreography.ChoreographyService.Tests/ChoreographyControllerFixture.cs

```diff
@@ -50,8 +54,12 @@ public async Task Post_Returns200_ForSubscriptionValidationData()
         public async Task Post_Returns400_ForInvalidSubscriptionValidationData()
         {
             var loggerMock = new Mock<ILogger<ChoreographyController>>();
-            var eventOp = new EventGridEvent();
-            eventOp.EventType = EventTypes.EventGridSubscriptionValidationEvent;
+            var eventOp = new EventGridEvent(
+                "EventSubject",
+                "EventType",
+                "1.0",
+                "This is the event data");
+            eventOp.EventType = "Microsoft.EventGrid.SubscriptionValidationEvent";
             eventOp.Data = null;
             var target = new ChoreographyController(new Mock<IPackageServiceCaller>().Object,
                                                 new Mock<IDroneSchedulerServiceCaller>().Object,
```

Comment:
nit:  This spacing is really weird.   We should either be doing a standard 4-space indent or aligning with the `new` above.   I'm going to leave this as the one comment, but there are several other instances in this file.

Reply:
I fixed all the format in this file with 4-space indent.